### PR TITLE
Migration of account-move-line-report-xls to V8

### PR DESCRIPTION
**Pull request by JehoG**

### CODE REVIEW

File: account_move_line_report_xls/account_move_line.py [hunk 1]
```diff
@@ -20,14 +20,15 @@
...
+from openerp import models, api
 
 
-class account_move_line(orm.Model):
+class account_move_line(models.Model):
     _inherit = 'account.move.line'
 
     # override list in custom module to add/drop columns or change order
-    def _report_xls_fields(self, cr, uid, context=None):
+    @api.multi
```
Comment (guewen) [inline account_move_line_report_xls/account_move_line.py]: It should be `@api.model`

File: account_move_line_report_xls/__init__.py [hunk 2]
```diff
@@ -21,8 +21,8 @@
 ##############################################################################
 
 try:
-    from . import account_move_line
-    from . import report
+    import account_move_line
+    import report
```
Comment (guewen) [inline account_move_line_report_xls/__init__.py]: Is there a specific reason to change that to an absolute import?
The relative import is safer and that's what should be used preferably.

File: account_move_line_report_xls/account_move_line.py [hunk 3]
```diff
@@ -40,7 +41,8 @@ def _report_xls_fields(self, cr, uid, context=None):
         ]
 
     # Change/Add Template entries
-    def _report_xls_template(self, cr, uid, context=None):
+    @api.multi
```
Comment (guewen) [inline account_move_line_report_xls/account_move_line.py]: `@api.model`

File: account_move_line_report_xls/report/__init__.py [hunk 4]
```diff
@@ -20,4 +20,4 @@
 #
 ##############################################################################
 
-from . import move_line_list_xls
+import move_line_list_xls
```
Comment (guewen) [inline account_move_line_report_xls/report/__init__.py]: again, safer to use relative import

**Comment by guewen** [comment]
Thanks for porting this module. I added notes on a few things that can be improved.

**Comment by JehoG** [comment]
Thanks for your comments. I changed those things.

**Comment by guewen** [comment]
:+1:

File: account_move_line_report_xls/report/__init__.py [hunk 5]
```diff
@@ -21,3 +21,4 @@
 ##############################################################################
 
 from . import move_line_list_xls
+
```
Comment (yvaucher) [inline account_move_line_report_xls/report/__init__.py]: Please don't add empty line at EOF

```
./account_move_line_report_xls/report/__init__.py:24:1: W391 blank line at end of file
```

File: account_move_line_report_xls/report/move_line_list_xls.py [hunk 6]
```diff
@@ -134,9 +134,9 @@ def __init__(self, name, table, rml=False, parser=False, header=True,
                 'header': [1, 13, 'text', _render("_('Maturity Date')")],
                 'lines':
                 [1, 0,
-                 _render("line.date_maturity.val and 'date' or 'text'"),
+                 _render("line.date_maturity and 'date' or 'text'"),
                  _render(
-                     "line.date_maturity.val \
+                     "line.date_maturity \
```
Comment (yvaucher) [inline account_move_line_report_xls/report/move_line_list_xls.py]: To put a string on multiple line you can write

```
("start of string"
 " continuing string"
 " end of string")
```

**Comment by JehoG** [comment]
@yvaucher Is this PR seems ok to you?

**Comment by yvaucher** [comment]
It would a good time to extract 'description' from `__openerp__.py` manifest to a `README.rst` file

And then :+1:

**Comment by yvaucher** [comment]
@JehoG Could you use the OCA template please, sorry I forgot to mention it:

https://raw.githubusercontent.com/OCA/maintainers-tools/master/template/module/README.rst

**Comment by JehoG** [comment]
Is this ok?
Not sure about the contributor, since there was no one there, if I add me it looks like i'm the only one to have contributed...

**Comment by yvaucher** [comment]
Thanks @JehoG

**Pull request merged**